xchange-stream-ftx/src/main/java/info/bitrich/xchangestream/ftx/FtxStreamingService.java

```diff
@@ -79,6 +79,8 @@ protected String getChannelNameFromMessage(JsonNode message) {
 
     if ("fills".equals(message.get("channel").asText())) {
       channelName = message.get("channel").asText();
+    } else if ("orders".equals(message.get("channel").asText())) {
```

Comment:
if this is the same have the if contain an || since the action is the same

Reply:
btw, why not adding sonar or something like that to the project?

Reply:
lol i just saw it @earce

Reply:
what is sonar @cymp ? thanks